subprojects/plugins/src/testFixtures/groovy/org/gradle/api/tasks/compile/AbstractCachedCompileIntegrationTest.groovy

```diff
@@ -25,18 +25,24 @@ abstract class AbstractCachedCompileIntegrationTest extends AbstractIntegrationS
         setupProjectInDirectory(temporaryFolder.testDirectory)
     }
 
+    def expectDeprecationWarnings() {
```

Comment:
We probably don't need this empty method

Reply:
It's a template method needed for the `CachedPlatformScalaCompileIntegrationTest` subclass.